lib/influx.js

```diff
@@ -0,0 +1,47 @@
+/* 
+* Reports request metrics to InfluxDB
+* @param {Request} request - incoming request
+* @param {object} env - environment variables
+*/
+export const reportRequestMetric = async (request, env) => {
```

Comment:
```suggestion
export const reportRequestMetrics = async (request, env) => {
```

The other functions also say it's metrics plural